pyexebuilder/ExeBuilder.py

```diff
@@ -51,20 +55,39 @@ def create_tmp_dir(self):
     def clean_up_tmp_dir(self, tdir):
         pass
 
+    def get_extra_relative_path_elements(self):
+        e = []
+        # pywintypes and pythoncom have weird contortions to find DLLs: (look at the source in pywintypes.py)
+        # They also do something different if 'frozen' - which is the case when running from the exe
+        # We need to add the pywin32_system32 folder to the pythonpath:
+        if self.include_pythoncom_and_pywintypes_in_pythonpath():
+            e.append("Lib\\site-packages\\pywin32_system32")
+        return e
+
```

Comment:
This was in an old commit and then removed. We actually do need this. I have no idea how the code was working before without this.

Reply:
So based on the previous commit, this was removed because it did not work. I think this may need some additional testing.